ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml

```diff
@@ -11469,7 +11469,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build01
-  cron: 0 4 * * 3
+  cron: 0 4 * * 1
```

Comment:
prow-config job is complaining about the timing

Reply:
The issue was on the build cluster provided it seems, changed it to build01 as suggested. Now it has passed